achievements/api-platform.md

```diff
@@ -1,51 +1,48 @@
-# API Platform Project achievements
+# API Platform Project Achievements
```

Comment:
I think we use 'achievements' in the other pages.

Reply:
I think this is better like this (https://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html), so I will do the modification for the other pages as well.

Reply:
I do not have a strong opinion on this, so would use the EC style guide as a reference: https://ec.europa.eu/info/sites/info/files/whats_new_style_guide_en.pdf
In 4.10, I see that titles take a capital only on the first word.